src/stores/order.store.ts

```diff
@@ -16,18 +20,29 @@ export enum ORDER_STATE {
 }
 
 export class OrderStore {
+  public static get instance(): OrderStore {
+    if (!this[singleton]) this[singleton] = new OrderStore(singletonEnforcer)
+    return this[singleton]
+  }
+
   private openOrders: Map<string, Map<string, OrderWithTrades>> = new Map()
   private orderStates: Map<string, Map<string, ORDER_STATE>> = new Map()
-  private readonly sessionId = sessionStore.sessionId
+  private readonly sessionId = SessionStore.instance.sessionId
+
+  constructor(enforcer: Symbol) {
```

Comment:
Mark constructor as private.
Can you also do this for every other store? then i have not to copy the comment. :-)